go.mod

```diff
@@ -2,10 +2,12 @@ module github.com/foxcpp/filedrop
 
 require (
 	github.com/go-sql-driver/mysql v1.4.0
-	github.com/gofrs/uuid/v3 v3.1.1
+	github.com/gofrs/uuid v3.2.0+incompatible
 	github.com/lib/pq v1.0.0
 	github.com/mattn/go-sqlite3 v1.9.0
 	github.com/pkg/errors v0.8.0
 	google.golang.org/appengine v1.2.0 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 )
+
+go 1.13
```

Comment:
It is annoying, can you remove it?  I am not sure if Go 1.11 ignores it. Go 1.12 will emit a warning that really makes no sense since we use no features specific to Go 1.13.

Reply:
I did not do that by myself, sorry. I will remove that and ignore the file locally for the future.